Update ResolveQueryData with info about resolving uninitialized queries

john-h-k opened the pull request:
based on comments in the DX discord

john-h-k committed:
Update ResolveQueryData with info about resolving uninitialized queries

based on comments in the DX discord

PRMerger20 commented:
@john-h-k : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edit pass

Note that it's correct to remove `/en-us/` from urls. It's also correct to remove `https://docs.microsoft.com`, for folks using the offline Microsoft Help Viewer.

PRMerger17 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
